src/benchmarks/real-world/Microsoft.ML.Benchmarks/HashBench.cs

```diff
@@ -17,6 +18,7 @@ namespace Microsoft.ML.Benchmarks
     /// The benchmarks initialize the hashing transformer with a 20 bit hash, then measure
     /// perf on fetching the first 100,000 values.
     /// </summary>
+    [BenchmarkCategory(Categories.MachineLearning)]
```

Comment:
same as above: we don't need categories if there is only one category in given project

Reply:
See above.